common/src/main/java/com/wynntils/mc/event/PlayerGhostArmorRenderEvent.java

```diff
@@ -0,0 +1,29 @@
+package com.wynntils.mc.event;
+
+import com.wynntils.core.WynntilsMod;
+import net.minecraft.world.entity.LivingEntity;
+import net.minecraftforge.eventbus.api.Event;
+
+public class PlayerGhostArmorRenderEvent extends Event {
+
+    private LivingEntity entity;
+    private boolean renderGhostArmor;
```

Comment:
remove this

Reply:
I don't see why?

Reply:
we should use cancelable like everywhere else

Reply:
Also, can you rename the event PlayerArmorRenderEvent? That is more appropriate